src/js/events.js

```diff
@@ -487,7 +487,7 @@ var Events;
         handleKeydown: function (event) {
             this.triggerCustomEvent('editableKeydown', event, event.currentTarget);
 
-            if (Util.isKey(event, Util.keyCode.ENTER)) {
+            if (Util.isKey(event, Util.keyCode.ENTER) || Util.isKey(event, Util.keyCode.ENTER_OF_EMACS)) {
```

Comment:
Yes, <kbd>CTRL</kbd> + <kbd>M</kbd>.
So I add the check for CTRL.

Thank you sample code. But the sample code couldn't well, because I use Mac. So I'll use <code>event.ctrlKey</code>.

```javascript
if (Util.isKey(event, Util.keyCode.ENTER) || (event.ctrlKey && Util.isKey(event, Util.keyCode.M))) {
    // ...
}
```